static/css/restyle.less

```diff
@@ -96,8 +96,9 @@ h2 {
 h1,
 .primary h2,
 hgroup h2.addon,
-.personas-featured h3 {
-  color: @default-font-color;
+.personas-featured h3,
+#featured-addons .featured-inner h3 {
+  color: @header-font-color;
```

Comment:
These headeings are supposed to be #333 - which is getting removed with this change by the looks of it.